lib/components/primitive-components/Group/Group.ts

```diff
@@ -300,6 +300,10 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
       }
 
       if (job.has_error) {
+        db.pcb_autorouting_error.insert({
+          pcb_error_id: autorouting_job.autorouting_job_id,
+          message: job.error!,
+        })
         throw new Error(`Autorouting job failed: ${JSON.stringify(job.error)}`)
```

Comment:
nice! We can _probably_ remove this error throw but idc if it's not causing problems atm

`job.error` isn't a string though, you need to stringify it or probably do `job.error?.message ?? JSON.stringify(job.error)`